src/components/Table/Cell.js

```diff
@@ -98,7 +98,7 @@ const Cell = (props) => {
   };
 
   const wrapChildren = (content) => {
-    if (!!sortable && isHeader) {
+    if (isHeader) {
```

Comment:
Shouldn't this return a span in case it is not a an action like `sortable`?

Reply:
It breaks visual styles for non-sortable fields